packages/clerk-js/src/ui/contexts/ClerkUIComponentsContext.tsx

```diff
@@ -510,7 +510,8 @@ export const useGoogleOneTapContext = () => {
     options,
     {
       ...ctx,
-      redirectUrl: window.location.href,
+      signInFallbackRedirectUrl: window.location.href,
+      signUpFallbackRedirectUrl: window.location.href,
```

Comment:
@nikosdouvlis is this change ok ? i wasn't sure if i needed to use the fallback or force urls